src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -112,7 +112,7 @@ private Tiktoken(Stream vocabStream, IReadOnlyDictionary<string, int>? specialTo
         /// <param name="cacheSize">The size of the cache to use.</param>
         /// <param name="normalizer">To normalize the text before tokenization</param>
         /// <returns>The tokenizer</returns>
-        public static Tokenizer CreateByModelName(
+        public static Tokenizer CreateTokenizerForModel(
```

Comment:
Please make sure we help existing folks react to this breaking change once we make it.  We may want to have them wait until we take this through API review which may have other changes.

Reply:
Yes, we need to keep track of all parties and we'll help communicate the breaking changes once we design review it.